src/pipx/pipx_metadata_file.py

```diff
@@ -35,9 +35,13 @@ class PackageInfo(NamedTuple):
     apps_of_dependencies: List[str]
     app_paths_of_dependencies: Dict[str, List[Path]]
     package_version: str
+    suffix: str = ""
```

Comment:
I think `suffix` should be a member of `PipxMetadata`.  I don't think we want more than one suffix for a given venv.